src/app/shared/register-github-app/register-github-app.component.html

```diff
@@ -29,16 +29,17 @@
     >.
   </p>
   <div fxLayout="column" fxLayoutAlign="center center" fxLayoutGap="2rem" class="mb-4">
-    <a
+    <button
       mat-raised-button
       type="button"
       class="accent-1-dark-btn mat-elevation-z no-underline"
-      [href]="gitHubAppInstallationLink$ | async"
+      (click)="openGitHubApp()"
```

Comment:
The anchor tag does not have a disabled attribute (line 39), Angular 15 started error on this, so changed to a button, which doesn't have an href. This covers the changes in the controller as well.